junit-jupiter-api/src/templates/resources/main/org/junit/jupiter/api/condition/JRE.java.jte

```diff
@@ -193,7 +150,13 @@ public enum JRE {
 	 */
 	@API(status = STABLE, since = "5.12")
 	public static JRE currentJre() {
```

Comment:
Regarding the Javadoc for `currentVersion()`, similar to what I stated in another comment, I don't think that the _current_ `JRE` can technically be `UNDEFINED` anymore.

Reply:
Done in 9ac1c76da531ab463952f1951409581fbb9a052e